Common/cpp/Tools/JSISerializer.cpp

```diff
@@ -53,39 +51,38 @@ std::string stringifyJSIFunction(jsi::Runtime &rt, const jsi::Function &func) {
 std::string stringifyJSIHostObject(
     jsi::Runtime &rt,
     jsi::HostObject &hostObject) {
-  std::stringstream ss;
   int status = -1;
   const char *hostObjClassName =
       abi::__cxa_demangle(typeid(hostObject).name(), NULL, NULL, &status);
-  if (status == 0) {
-    auto props = hostObject.getPropertyNames(rt);
-    auto propsCount = props.size();
-    auto lastKey = props.back().utf8(rt);
-
-    ss << '[' << hostObjClassName << ' ';
-    if (propsCount > 0) {
-      ss << '{';
-      for (auto &key : props) {
-        auto formattedKey = key.utf8(rt);
-        facebook::jsi::Value value = hostObject.get(rt, key);
-        ss << '"' << formattedKey << '"' << ": "
-           << reanimated::stringifyJSIValue(rt, value);
-        if (formattedKey != lastKey)
-          ss << ", ";
+  if (status != 0) {
+    return "[jsi::HostObject]";
+  }
+
+  std::stringstream ss;
+  auto props = hostObject.getPropertyNames(rt);
+  auto propsCount = props.size();
+  auto lastKey = props.back().utf8(rt);
+
+  ss << '[' << hostObjClassName << ' ';
+  if (propsCount > 0) {
+    ss << '{';
+    for (auto &key : props) {
+      auto formattedKey = key.utf8(rt);
+      facebook::jsi::Value value = hostObject.get(rt, key);
+      ss << '"' << formattedKey << '"' << ": "
+         << reanimated::stringifyJSIValue(rt, value);
```

Comment:
Is it possible to call `stringifyJSIValue`, i.e. without `reanimated::` everywhere?